### ISSUE TITLE
Trouble with calling "fontforge -usage/-help/--help"

### ISSUE THREAD

**Issue by hartwork** [hanwen/mftrace#2]
Some versions of fontforge (e.g. the latest version 20120731) open a web browser not just for `-usage` but for `-help`, too. Despite their docs saying otherwise. Proof:

```
# strace -eprocess -F fontforge -help |& fgrep xdg-open
[pid  3966] execve("/bin/sh", ["sh", "-c", "\"xdg-open\" \"http://fontforge.sf."...], [/* 63 vars */]) = 0
execve("/usr/bin/xdg-open", ["xdg-open", "http://fontforge.sf.net/overview"...], [/* 61 vars */]) = 0

# fontforge -version
Copyright (c) 2000-2012 by George Williams.
 Executable based on sources from 14:57 GMT 31-Jul-2012.
 Library based on sources from 14:57 GMT 31-Jul-2012.
fontforge 20120731
libfontforge 20120731
```

While you could argue that the bug should be fixed in fontforge, there's an easy fix to make mftrace work for users of that fontforge version, too: I propose to turn the fontforge command detection from calling `fontforge -usage` to calling `fontforge -version`. Many thanks for your consideration.

**Comment by hartwork** [comment]
Please fix, it's three years soon :)

**Comment by hanwen** [comment]
seems to be fixed now (fontforge 20170308).

**Comment by hartwork** [comment]
Hi!

I fail to find release 20170308 that you mention at https://github.com/fontforge/fontforge/tags . Where did you see it?

Since distros don't have FontForge that new and may also bump mftrace quicker than FontForge, could you still do that tiny change to calling `fontforge -version` rather than `fontforge -usage`?

Best, Sebastian
